pkg/client/actionclient.go

```diff
@@ -205,6 +254,25 @@ func (c *actionClient) Install(name, namespace string, chrt *chart.Chart, vals m
 	}
 	install.ReleaseName = name
 	install.Namespace = namespace
+
+	// TODO: Open Question -  should we always run preflight checks if they exist?
```

Comment:
Open to suggestions here.

My thinking behind not running the preflight checks if dry run is specified is because the caller may have a certain expectation of the output with a dry run and including preflight checks in that could pollute the dry run results.